foundry.md

```diff
@@ -96,8 +96,14 @@ First we have some helpers in K which can:
              ...
          </account>
 ```
-```k
 
+#### `deal` - Set a given balance to a given account.
+
+```
+function deal(address who, uint256 newBalance) external;
+```
```

Comment:
This isn't documentation, it's a function signature. Please put some more information here (a sentence or two) along with the function signature abotu what `deal` does for the user.